src/grp_uvx.py

```
@@ -252,15 +252,16 @@ def extract_problem_description(body: str) -> str:
 
 
 def generate_coderabbit_curl_commands_for_comment(owner: str, repo: str, pr_number: int, comment_id: int, token: str) -> str:
-    """特定のコメントに対するCodeRabbit返信用のcurlコマンドを生成"""
+    """特定のコメントに対するCodeRabbit返信用のcurlコマンドを生成（3パターンのみ）"""
     templates = {
-        "対応不要": f"@coderabbitai この指摘について確認しましたが、[技術的根拠]により対応不要と判断します。問題がなければこの課題を解決済みにしてください。ただし、この課題のみを解決済みにし、他の課題をすべて解決済みにしないよう注意してください。",
-        "対応完了": f"@coderabbitai ご指摘いただいた点を修正しました。[修正内容]を実施済みです。問題がなければこの課題を解決済みにしてください。ただし、この課題のみを解決済みにし、他の課題をすべて解決済みにしないよう注意してください。",
+        "対応不要": f"@coderabbitai この指摘について確認しましたが、[技術的根拠]により対応不要と判断します。この課題のみを解決済みにしてください。",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)